features/FxA-56-email-deliverability/README.md

```diff
@@ -1,66 +1,160 @@
-# Email deliverability (FxA-56)
+# Email Deliverability (FxA-56)
 
 Feature card: https://github.com/mozilla/fxa-features/issues/56
 
 ## Problem Summary
 
-The Firefox Accounts email deliverability is suffering from
-lack of understanding of the common causes of email bounces. There are not enough metrics to detect email bounce reasons and understand the cause of issues.
+Users regularly report email deliverability issues with Firefox Accounts,
+and we do not have sufficient visibility into
+the common causes of email bounces
```

Comment:
Complaints are probably a good thing to keep track of, but it'd probably be odd if a user complained about an email and then wondered where it was. I suppose complaints could possibly lower our score with something like spamhaus or spamassassin.